commcare_connect/form_receiver/tests/test_receiver_integration.py

```diff
@@ -596,6 +587,44 @@ def test_reciever_verification_flags_catchment_areas(
     assert ["catchment", "Visit outside worker catchment areas"] in visit.flag_reason.get("flags", [])
 
 
+@pytest.mark.parametrize("opportunity", [{"managed": True}], indirect=True)
+def test_receiver_auto_agree_approved_visit(
```

Comment:
This could potentially be combined with `test_receiver_flagged_visit_review_pending` using parametrize

Reply:
Fixed.